client/packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -212,7 +212,6 @@ const DataTableComponent = <T extends RecordWithId>({
       sx={{
         display: 'flex',
         flexDirection: 'column',
-        height: '100%',
```

Comment:
offroad - this was changed in a derived project and it made me question why it was here. have found no impact.